### PR TITLE
Fixed broken links pointing to master branch

### PR DESCRIPTION
**Pull request by svick**

### CODE REVIEW

File: src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs [hunk 1]
```diff
@@ -64,7 +64,7 @@ private string PreconditionsMessage()
             log.Append(
                 "Incremental compilation will be automatically enabled if the above mentioned project properties are not used. " +
                 "For more information on the properties and how to address them, please consult:\n" +
-                @"https://github.com/dotnet/cli/blob/master/Documentation/addressing-incremental-compilation-warnings.md");
+                @"https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/addressing-incremental-compilation-warnings.md");
```
Comment (eerhardt) [inline src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs]: Hmm, this one concerns me.  I don't think we should be compiling hard-coded URLs in our code, especially pointing to branches in github.

@blackdwarf @cdmihai @piotrpMSFT - thoughts?  Should we make an fwlink for this?

**Comment by eerhardt** [comment]
Thanks @svick for finding these.  Let's get @blackdwarf's opinion on that URL in our code.  I think we will want to make it a different URL, one we can change without having to re-compile the code.

File: src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs (hunk 1 above)
Comment (blackdwarf) [inline src/dotnet/commands/dotnet-build/IncrementalPreconditions.cs]: @eerhardt eeek, yes, we should. aka.ms links also work. Also, I believe I've added these to the official docs on dotnet.github.io (https://dotnet.github.io/docs/core-concepts/core-sdk/cli/dotnet-build.html), so I will create these aka.ms links and let's replace the above with aka.ms ones. Will paste them as soon as I can get 2-phase auth to work. 😄

**Comment by blackdwarf** [comment]
OK, the link is created at https://aka.ms/dotnet-build. I went ahead and created aka.ms links for ALL THE COMMANDS. 😄 They are all in the shape above. Seemed like a good idea.

**Comment by svick** [comment]
@blackdwarf Updated.

**Comment by eerhardt** [comment]
@svick - the "rel/1.0.0" branch is currently broken.  It has a failing test - which you are running into.  There is no urgency in getting this in "rel/1.0.0", so let's wait for the test to get fixed and we'll re-run. It may be a few days though, we are busy with "preview2" work.

@blackdwarf @piotrpMSFT - Do we think this hard-coded URL meets the "preview2" bar?  Should we try porting that change to "1.0.0-preview2"?

**Comment by blackdwarf** [comment]
@eerhardt we should port. Is it a huge risk?

**Comment by eerhardt** [comment]
>  Is it a huge risk?

I think that it is very low risk.  Just changing one URL for another.

**Comment by blackdwarf** [comment]
@eerhardt let's port this change then pls. It will help us in the long run.

File: tools/DebianPackageTool/README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ directory structure with your desired files to be included.
 Note: The default install root is `/usr/share/{package_name}` where package_name is replaced with the name of the created package
 
 ## full example debian_config.json
-Note: Use the commentless version here (https://github.com/dotnet/cli/blob/master/packaging/debian/package_tool/example_config.json)
+Note: Use the commentless version here (https://github.com/dotnet/cli/blob/rel/1.0.0/tools/DebianPackageTool/example_config.json)
```
Comment (eerhardt) [inline tools/DebianPackageTool/README.md]: Is it possible to make a relative path here?

**Comment by eerhardt** [comment]
@svick - How do you want to handle the first file?  Do you just want to take what I merged into the `preview2` branch?

The 2nd file still needs to be fixed, so leaving this PR open for that.

File: tools/DebianPackageTool/README.md (hunk 2 above)
Comment (svick) [inline tools/DebianPackageTool/README.md]: Done.

**Comment by svick** [comment]
@eerhardt I slightly prefer this version, since it's one less line. But I don't feel strongly either way.

**Comment by eerhardt** [comment]
@svick - our merge of the preview2 branch came in and now this change needs to be rebased. :disappointed:   (what a lot of work for 2 simple string changes! sorry about that.)

After the change is rebased and green, I'll merge this is as quickly as I see it.

Thanks again for the contribution.

**Comment by svick** [comment]
@eerhardt Done.

**Comment by eerhardt** [comment]
:shipit:

**Comment by eerhardt** [comment]
test Windows_NT x64 Release Build

**Comment by eerhardt** [comment]
Thanks, @svick!

**Pull request merged**